.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+* **Please check if the PR fulfills these requirements**
+- [ ] The commit message follows our guidelines
+- [ ] **Tests** for the changes have been added (for bug fixes / features)
+- [ ] **Docs** have been added / updated (for bug fixes / features)
+- [ ] **Changelog** was updated
+
+* **Is this PR related to any issue?**
+(if so, reference the issue on the commit message or on the PR description.
+Also, if this PR closes/fixes the issue, also add `Fix #<issueNumber>` to close
+the issue automatically when the PR is accepted)
+
+* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
+
+
+* **What is the current behavior?** (You can also link to an open issue here)
+
+
+* **What is the new behavior (if this is a feature change)?**
```

Comment:
Can we ignore the 3 questions above could when there's a linked issue?

Reply:
Nice idea!